dynamicio/mixins.py

```diff
@@ -30,13 +30,18 @@
 from kafka import KafkaProducer  # type: ignore
 from magic_logger import logger
 from pyarrow.parquet import read_table, write_table  # type: ignore # pylint: disable=no-name-in-module
-from sqlalchemy import create_engine  # type: ignore
+from sqlalchemy import Boolean, Column, Float, Integer, String, create_engine  # type: ignore
+from sqlalchemy.ext.declarative import declarative_base  # type: ignore
 from sqlalchemy.orm import Query  # type: ignore
+from sqlalchemy.orm.decl_api import DeclarativeMeta  # type: ignore
 from sqlalchemy.orm.session import Session as SqlAlchemySession  # type: ignore
 from sqlalchemy.orm.session import sessionmaker  # type: ignore
 
 Session = sessionmaker(autoflush=True)
 
+Base = declarative_base()
+_type_lookup = {"boolean": Boolean, "object": String(64), "int64": Integer, "float64": Float, "int": Integer}
```

Comment:
So, it seems like we have an issue here. We are missing a type and that's `DateTime`. 

I think it needs to be added. The error I got from trying the release candidate was not very clear, as it was a key-error. But I think this is it.

Reply:
I am pretty sure this is the problem. I can confirm that I was able to read other tables which did not have that dtype. 

Still to confirm we can write. Let's get this fixed and we can try again.

Reply:
FYI @KitBurgess @hopeswiller

Reply:
Code broke here: https://github.com/miodo/dynamicio/blob/enhancement/implicit_read_model/dynamicio/core.py#L268 

with a key-error for a column that has dateime as the type.

Reply:
Please add a test to confirm we can use all dtypes as welll.

Reply:
In that case I suppose the type specified in the scheme definition yaml file was either timestamptz or date.
If not can you please confirm what the type was

Reply:
<img width="215" alt="Screenshot 2022-07-06 at 15 01 47" src="https://user-images.githubusercontent.com/50640801/177568410-e1a7c1b3-a1e0-497e-8182-c521db78a535.png">

Reply:
Assume a yaml schema as follows: 
<img width="278" alt="Screenshot 2022-07-06 at 15 05 09" src="https://user-images.githubusercontent.com/50640801/177569199-78f6a782-6705-45b9-ad61-48aea8d6e874.png">

Which had tobe converted as per below:
<img width="367" alt="Screenshot 2022-07-06 at 15 03 53" src="https://user-images.githubusercontent.com/50640801/177568899-a4727d93-5458-42b7-ab54-f39bf6b15c4f.png">